README.md

```diff
@@ -17,16 +17,33 @@ Tour of the source
 * `robotd/devices_base.py` contains some common code for `devices.py`.
 * `robotd.service` is the systemd service which runs the thing in production.
 
-Building the Debian package
+Getting started
+---------------
+
+Since `robotd` vendors in April Tags for its vision support, which depends on
+`libusb`, you'll need to install the development package for `libusb` in order
```

Comment:
Is this correct? From https://github.com/sourcebots/sb-vision/blob/master/debian/control (no reference to libusb) and https://github.com/sourcebots/sb-vision/blob/807477ffdad87228be50475a8c84c1ecf94b325a/sb_vision/native/apriltag/contrib/april/README ("no external dependencies"), it seems like `libusb` isn't required for April Tags or the `sb-vision`.

Reply:
I think the statement that AprilTags needs `libusb` is actually wrong. `robotd` depends on `libusb` directly.

Reply:
I don't know whether we need `libusb`'s dev package any more.